styles.css

```diff
@@ -1,590 +1,1169 @@
+    color: #172b4d;
+    padding-top: 2px;
+    line-height: 44px;
+    margin-top: 3px;
+    margin-bottom: 9px;
+  }
+
+  .first-item {
+    display: flex;
+    flex-direction: row;
+    padding-inline-start: 0;
+  }
+
+  .one {
+    list-style-type: none;
+    color: #344563;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .other {
+    font-weight: 600;
+    color: #7a869a;
+    font-size: 0.9rem;
+    padding-right: 8%;
+  }
+
+  .text-1 {
+    display: inline-flex;
+    flex-direction: column;
+    color: #344563;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+  }
+
+  .item-2 li {
+    list-style-type: none;
+    border-radius: 20px 20px;
+    line-height: 15px;
+  }
+
+  .link-item {
+    display: inline-block;
+    background: #ebebff;
+    border-radius: 0.5rem;
+    padding-left: 5%;
+    padding-right: 5%;
+    padding-top: 2%;
+    padding-bottom: 2%;
+    font-weight: 500;
+    font-size: 0.75rem;
+    color: #6070ff;
+  }
+
+  .see {
+    font-size: 18px;
+    font-weight: 400;
+    border: 2px solid #396df2;
+    color: #396df2;
+    width: 40%;
+    border-radius: 10px 12px;
+    height: 30px;
+    padding-left: 20px;
+    padding-right: 20px;
+    padding-top: 9px;
+    margin-left: 5px;
+    margin-right: 5px;
+  }
+
+  .see:hover {
+    background-color: #d1d5df;
+  }
+
+  .see-active {
+    font-size: 18px;
+    font-weight: 400;
+    border: 2px solid #396df2;
+    color: #fff;
+    width: 40%;
+    border-radius: 10px 12px;
+    height: 30px;
+    padding-left: 20px;
+    padding-right: 20px;
+    padding-top: 9px;
+    margin-left: 5px;
+    margin-right: 5px;
+    background-color: #396df2;
+  }
+
+  .resume {
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    color: #396df2;
+    background: #fff;
+    height: 50px;
+    padding-left: 3%;
+    margin-left: 2%;
+  }
+
+  button {
+    width: 50%;
+    border: 1.5px  solid #396df2;
+    border-radius: 10px 10px;
+    margin-bottom: 40px;
+  }
+
+  .resume:active {
+    background-color: #fff;
+  }
+
+  .resume:hover {
+    background-color: #637fc4;
+  }
+
+  .lang {
+    padding-left: 5px;
+    padding-top: -1px;
+    color: #000;
+    font-weight: 500;
+    line-height: 24px;
+    font-style: normal;
+    font-size: 20px;
+  }
+
+  .langs {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .about-me {
+    padding-bottom: 1px;
+    background-color: #ffff;
+    width: inherit;
+  }
+
+  .abt-self {
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    padding-top: 8%;
+    text-align: left;
+    color: #091e42;
+  }
+
+  .seperate-1 {
+    padding-left: 6%;
+    padding-right: 2%;
+  }
+
+  .seperate-2 {
+    padding-top: 3.5%;
+    padding-left: 2%;
+    padding-right: 6%;
+  }
+
+  .works1 {
+    display: flex;
+    flex-direction: row;
+    padding-top: 4%;
+    padding-bottom: 6%;
+  }

```

Comment:
- [ ]  Please make sure to adhere to the `Figma` design template as much as possible as currently, your design doesn't match the `Figma` design given. This will make your design look beautiful and create a good user experience. Kindly consider the suggestions below to help improve your design:
    - [ ] Kindly style your `about section` properly such that the first section which talks about you and your social media icons and others has a smaller portion of the container as compared to that of the languages section. 
    - [ ] All contents of your about section should be aligned on the `left` to match that of the `Figma` design.
    - [ ] Kindly styles your **Get my resume** `buttons` to match that of the `Figma` design and also remember to add a `:hover` effect using the colors provided on `Figma`.
    - [ ] It would be better if you adjust the padding around the languages section to match that of the `Figma` design as it's currently not.
    - [ ] Please make sure that your `about section` is consistent across all screen sizes ( `mobile`, `table`, and `desktop`. See below for more details: 
     <table>
<tr>
<th align=“middle”>Desktop: Figma design</th>
<th align=“middle”>Desktop: Your design</th>
</tr>
<tr>
<td>
<img width="1440" alt="Screenshot 2023-05-13 at 01 03 54" src="https://github.com/hetrox8/portfolio1/assets/107192076/7ebe4aa4-803e-41b2-b45f-c7887f122a97">
</td>
<td>
<img width="1440" alt="Screenshot 2023-05-13 at 00 58 55" src="https://github.com/hetrox8/portfolio1/assets/107192076/2c3b8e7f-0247-494f-8210-41088f44563c">
</td>
</tr>
<tr>
<th align=“middle”>Mobile: Figma design</th>
<th align=“middle”>Mobile: Your design</th>
</tr>

<tr>
<td>
<img width="372" alt="Screenshot 2023-05-13 at 01 05 56" src="https://github.com/hetrox8/portfolio1/assets/107192076/2876537c-8925-4d5c-a437-a0c666b8d986">
</td>
<td>
<img width="372" alt="Screenshot 2023-05-13 at 01 04 45" src="https://github.com/hetrox8/portfolio1/assets/107192076/fdb7e329-2105-44f9-9dfd-7c52ce75046b">
</td>
</tr>
</table>